### ISSUE TITLE
Issue using 2.1.3.0-RC15 with ZIO 1.0.0-RC21

### ISSUE THREAD

**Issue by nbenns** [zio/interop-cats#197]
I assume this is due to fibre supervision changes:

```
Exception in thread "zio-default-async-4-667346055" java.lang.NoSuchMethodError: 'void zio.ZIO$Fork.<init>(zio.ZIO)'
	at zio.interop.CatsConcurrentEffect.$anonfun$runCancelable$2(cats.scala:204)
	at zio.Runtime.unsafeRunAsyncCancelable(Runtime.scala:125)
	at zio.Runtime.unsafeRunAsyncCancelable$(Runtime.scala:105)
	at zio.Runtime$$anon$3.unsafeRunAsyncCancelable(Runtime.scala:247)
	at zio.Runtime.unsafeRunAsync(Runtime.scala:94)
	at zio.Runtime.unsafeRunAsync$(Runtime.scala:93)
	at zio.Runtime$$anon$3.unsafeRunAsync(Runtime.scala:247)
	at zio.interop.CatsConcurrentEffect.$anonfun$runCancelable$1(cats.scala:206)
	at cats.effect.internals.IORunLoop$.step(IORunLoop.scala:193)
	at cats.effect.IO.unsafeRunTimed(IO.scala:327)
	at cats.effect.IO.unsafeRunSync(IO.scala:246)
	at cats.effect.SyncIO.unsafeRunSync(SyncIO.scala:51)
	at org.http4s.server.blaze.Http1ServerStage$$anon$2.run(Http1ServerStage.scala:186)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

**Comment by ghostdogpr** [comment]
Yes, ZIO releases pre-1.0 are not binary compatible, so you need a version of interop-cats built for RC21. There’s already a PR for it https://github.com/zio/interop-cats/pull/195

**Comment by nbenns** [comment]
I just found it thanks.
I'll defer the upgrade to zio until its released.

**Comment by nbenns** [comment]
Closing as PR #195 is now merged and in release `2.1.3.0-RC16`.
